platforms/web/web.js

```diff
@@ -0,0 +1,181 @@
+"use strict";
+
+const Bluebird = require("bluebird");
+const { ButtonsListMessage, Button, TextMessage, ButtonsMessage} = require("../generics");
+const WebMessage = require("../../models/web.model");
+const config = require("../../config/config-loader").load();
+const { camelCase } = require("lodash");
+
+
+function getTextMessage(message) {
+  return {
+    message,
+    buttons: []
+  };
+}
+
+function getButton(button) {
+  return {
+    message: null,
+    buttons: [
+      button
+    ]};
+}
+
+function getListButton(buttonList) {
+  return {
+    message: buttonList.text,
+    buttons: buttonList.attachments.buttons || []
```

Comment:
buttonList is always an instance of ButtonsListMessage or ButtonsMessage (platforms/generics), attachements is always defined :)